modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,24 @@
 components:
     schemas:
+        BatchEngineJobResponse:
+            properties:
+                batchId:
```

Comment:
I think this naming is not very correct. In order to check the status of the task, Liferay will probably use the `/o/headless-batch-engine/v1.0/import-task/{importTaskId}'  endpoint, so in our opinion, the name should be something like `importTaskId`, `batchImportTaskId`, etc